### DISCUSSION TITLE
Detected blocking call to sleep

### DISCUSSION THREAD

**Discussion by ops244** [make-all/tuya-local#1614] [General]
hi, there are 5 identical devices, 4 are connected by ip, 1 is not. What could be the problem? everyone's IP is fixed
and it's not the only one. there are 5 switches - 4 are connected, 1 is not and there are 3 remotes - 2 are connected, 1 is not)

**Comment by make-all** [comment]
The blocking call to sleep happens inside tinytuya when you did not specify an IP address, as it needs to wait for broadcast advertisements from the device.

**Comment by ops244** [comment]
figured it out)) thanks
